src/directives/justgage.js

```diff
@@ -85,6 +103,17 @@ angular.module('frapontillo.gage.directives', ['frapontillo.gage.controllers'])
                 justgage.refresh(scope.value, newValue);
               }
             }));
+            
+            // will fire only if the attribute 'watch-view-port' is set to true
+            // if fired, justgage element is brought back into view
+            // if something slides it off the screen or sets ng-show/hide this will ensure the justgage is redrawn and animated properly
+            if (justgageCtrl.getOptionValue('watchViewPort', scope.watchViewPort) === true) {
+              watchers.push(scope.$watch(function () { return isElementInViewPort(element[0]); }, function (newValue) {
```

Comment:
By removing the `el` argument to the function `isElementInViewPort` you can pass it as a simple argument to `$watch`.